tests/test_sensitivity_matrix.py

```diff
@@ -65,19 +65,67 @@ def setUp(self):
             # which we have excluded from the unrotated_sensitivity_matrix.
             self.gq_sensitivity_matrix = self.gq_sensitivity_matrix[:31, ...]
 
+            # Because the old sensitivity matrix was in the Zemax Coordinate
+            # System we need to swap the following zernikes:
```

Comment:
I thought you meant the sensitivity matrix had been updated to have the consistent coordinate system as others at the meeting at the last week?

If this is not done yet, do you have another ticket to do so? If yes, please help to put the ticket number here. We would need to have a plan to unify the coordinate system for all the policy files in this repository to make the future's maintenance easier.

Reply:
That's right. This swap here is performed to check it against the old sensitivity matrix which did not have the same coordinate system. Therefore the old sensitivity matrix needs to be swaped to match the new one. We could also remove this test, but I think it's useful.